src/base-tailwind.css

```
@@ -0,0 +1,95 @@
+
+.dokan-layout {
+    @import "tailwindcss/preflight.css" layer(base);
+    @import "tailwindcss/utilities.css";
+
+    /* Import custom component styles */
+    @import './dokan-components.css';
+
+    border-color: var(--color-border);
+
+}
+@import '@getdokan/dokan-ui/dist/dokan-ui.css';
```

Biome (2.1.2)

[error] This `@import` is in the wrong position.

Any `@import` rules must precede all other valid at-rules and style rules in a stylesheet (ignoring `@charset` and `@layer`), or else the `@import` rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)